src/cljs/main/broadfcui/page/workspace/create.cljs

```diff
@@ -61,11 +62,10 @@
              {:text [:div {} [:strong {} "Note:"]
                      [:div {} "An Authorization Domain can only be set when creating a Workspace.
                    Once set, it cannot be changed."]
-                     (style/create-link {:href "https://software.broadinstitute.org/firecloud/documentation/article?id=9524"
-                                         :target "_blank"
-                                         :text [:span {:style {:white-space "pre"}}
-                                                "Read more about Authorization Domains"
-                                                icons/external-link-icon]})]})]
+                     (links/create-external
+                      {:href "https://software.broadinstitute.org/firecloud/documentation/article?id=9524"}
+                      [:span {:style {:white-space "pre"}}
```

Comment:
This `span` should be around the whole link so the external link icon isn't lost.